index.js

```diff
@@ -69,10 +69,26 @@ function join(array, string) {
   //your code here
 }
 
+// Now let's make something new entirely! We're going to make filterMap.
+//
+// This is a function which takes an array, a function which returns a boolean,
+// and a function which maps the array element.
+//
+// It should be functionally the same as array.filter(filterFn).map(mapFn)
+//
+// This is actually a useful function to write with reduce as it allows you to
```

Comment:
Maybe highlight that you could use this in production code
eg

Unlike the previous functions, this is actually a...